src/main/scala/com/databricks/spark/xml/XmlOptions.scala

```diff
@@ -76,6 +81,9 @@ private[xml] object XmlOptions {
   val DEFAULT_ROOT_TAG = "ROWS"
   val DEFAULT_CHARSET = "UTF-8"
   val DEFAULT_NULL_VALUE = null
+  val DEFAULT_TIMESTAMP_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS"
```

Comment:
@jameswinegar, shall we match this to CSV's in Spark? `yyyy-MM-dd'T'HH:mm:ss.SSSXXX`. IIRC, that complies ISO 8601 and should be good to be consistent with it.